### PR TITLE
feat(kit): `TUI_AVATAR_OPTIONS` default options configuration for `Avatar`

### PR DESCRIPTION
**Pull request by zarghamkhandev**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows [Conventional Commits](https://www.conventionalcommits.org/en/)
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [x] Feature
- [ ] Refactoring
- [ ] Code style update
- [ ] Build or CI related changes
- [ ] Documentation content changes

## What is the current behavior?

Partially implements https://github.com/Tinkoff/taiga-ui/issues/133

## What is the new behavior?

Default avatar options can be set via a provider. This is a partial implementation of https://github.com/Tinkoff/taiga-ui/issues/133

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: projects/kit/components/avatar/avatar-options.ts [hunk 1]
```diff
@@ -0,0 +1,29 @@
+import {InjectionToken, ValueProvider} from '@angular/core';
+import {TuiSizeXS, TuiSizeXXL} from '@taiga-ui/core/types';
```
Comment (nsbarsukov) [inline projects/kit/components/avatar/avatar-options.ts:2]:
```suggestion
import {TuiSizeXS, TuiSizeXXL} from '@taiga-ui/core';
```

File: projects/kit/components/avatar/tests/avatar-options.spec.ts [hunk 2]
```diff
@@ -0,0 +1,46 @@
+import {Component, ViewChild} from '@angular/core';
+import {ComponentFixture, TestBed} from '@angular/core/testing';
+import {
+    TuiAvatarComponent,
+    TuiAvatarModule,
+    tuiAvatarOptionsProvider,
+} from '@taiga-ui/kit';
```
Comment (nsbarsukov) [inline projects/kit/components/avatar/tests/avatar-options.spec.ts:7]: Declare them as relative imports

**Review by nsbarsukov — CHANGES_REQUESTED** [review]
Thanks for PR!

**Comment by splincode** [comment]
Hello, please, rebase with main branch before

File: projects/kit/components/avatar/tests/avatar-options.spec.ts (hunk 2 above)
Comment (zarghamkhandev) [reply projects/kit/components/avatar/tests/avatar-options.spec.ts]: done ✅

**Review by nsbarsukov — APPROVED** [review]

**Review by splincode — APPROVED** [review]

**Review by waterplea — APPROVED** [review]
Thank you for this PR!

**Pull request merged**